mrtt-ui/config-overrides.js

```diff
@@ -0,0 +1,15 @@
+const webpack = require('webpack')
+module.exports = function override(config) {
+  const fallback = config.resolve.fallback || {}
+  Object.assign(fallback, {
+    buffer: require.resolve('buffer/')
+  })
+  config.resolve.fallback = fallback
+  config.plugins = (config.plugins || []).concat([
+    new webpack.ProvidePlugin({
+      process: 'process/browser',
+      Buffer: ['buffer', 'Buffer']
+    })
+  ])
+  return config
+}
```

Comment:
Added this to fix build issues related to React 18 using Webpack 5. Node modules for jsZip which would be used on a node server were causing issues. The workaround was to modify the webpack config but because this application is created with create-react-app this would not usually possible without ejecting the app. This is one suggested workaround